content/docs/user-guide/networking/multiplayer/VersionMismatch.md

```diff
@@ -0,0 +1,30 @@
+# Multiplayer Version Mismatch
```

Comment:
H1 headers for titles, and other required information, need to go in to a [metadata section](https://www.o3de.org/docs/contributing/to-docs/style-guide/metadata/).
```suggestion
---
linkTitle: Version Mismatch
title: Multiplayer Version Mismatch
description: Learn how to detect multiplayer version mismatch in Open 3D Engine (O3DE).
weight: 900
---
```